This pull request is to merge restaurant-page-feature branch to development branch of the Javascript Restaurant-page project

henatan99 opened the pull request:
This project is building a static restaurant website with home, menu, and contact pages. The project employs webpack and ES6 module concepts to organize and compile the dependencies and modules into a distribution package.

henatan99 committed:
- Adds index.js file in src folder and index.html file in dist folder
- Links main.js in script tag in the index.html file
- Creates webpack.config.js file in the root directory
- Adds path variable and module.exports with entry and output attributes

henatan99 committed:
generates main.js in the dist folder using 'npx webpack' command and tests with a console.log message in the index.js file

henatan99 committed:
creates contactPage.js and menuPage.js module files in the src directory

henatan99 committed:
adds initialPage.js module file, style.css file and background.jpg file in src directory; imports style.css file into index.js file, and adds style-loader and css-loader in the webpack.config.js file and runs npx webpack

henatan99 committed:
adds file-loader using 'npm install --save-dev file-loader' command, and adds use: [file-loader] in module.rules of webpack.config.js file

henatan99 committed:
adds navBar.js file and adds contents to all .js files, import all the .js files into the index.js file append the dynamically created elements in the module files and adds stylings in the css file"

henatan99 committed:
adds menuItem function which creates a menu item div element with its image and item name contents
- adds food images in the src dirctory

henatan99 committed:
adds contents to menuPage module

henatan99 committed:
adds the contact page contents

henatan99 committed:
slight change on module files

MuriloRoque commented on package.json:
```diff
@@ -13,7 +13,13 @@
+    "webpack": "^5.24.4",
+    "webpack-cli": "^4.5.0"
+  },
   "scripts": {
```
- [ ] Add some additional scripts here. One is called "build", responsible for creating your dist folder with the resulting we pack compilation, another called "start", responsible for starting a local server on your computer with the compiled code. I believe you already ran these commands because you created a dist folder, but think about people who will use your app and want to clone your repository and make some changes for testing purpose, it would make their lives easier. Or maybe you will make some more changes yourself ☺️

MuriloRoque commented on src/index.js:
```diff
@@ -0,0 +1,35 @@
+content.appendChild(menuPage());
+content.appendChild(contactPage());
+
+// const nav = document.getElementsByClassName("nav-elem");
```
- [ ] Please remove this commented piece of code, it's not a good practice.

MuriloRoque reviewed (changes requested):
## Status: Changes requested :red_circle: 

### Hello @henatan99 :wave: 

### Great job on this project, everything is working as expected and your page is looking great, congratulations! There are only some issues we need to solve before moving on, alright? Please check my comments below and in your code.

- [ ] Add a nice screenshot to your README file, just so people can get a sneak peek at your page.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)

henatan99 committed:
Adds tabbed nav menu functionality logic in the index.js file usig click event listener

henatan99 committed:
adds contents to readme file

henatan99 committed:
fixes eslint errors

henatan99 committed:
fixes stylelint errors

mahmoud717 reviewed (approved):
### STATUS: APPROVED ✔️ ✔️ 

**hi @henatan99**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

henatan99 committed:
adds live link to readme file

henatan99 committed:
adds doc folder with screenshot images and adds screenshot demo imlinks in the readme file

henatan99 committed:
- adds "build" key in scripts of package.json file. It runs webpack bundler
- adds "start" key in scripts of package.json file. It opens local server
- installs 'webpack serve --open' using 'npm install --save-dev webpack-dev-server
' command

henatan99 committed:
deletes commented line fro index.js file

henatan99 committed:
fixes one eslintrc error
